apps/dashboard/src/components/share-buttom/index.tsx

```diff
@@ -2,9 +2,10 @@ import { Icon, useClipboard, useToast } from "@chakra-ui/react";
 import { FiCheck, FiShare2 } from "react-icons/fi";
 import { TrackedIconButton } from "tw-components";
 
-export const ShareButton: React.FC<Required<Omit<ShareData, "files">>> = (
-  shareData,
-) => {
+export const ShareButton: React.FC<ShareData> = (props) => {
+  const currentUrl = typeof window !== "undefined" ? window.location.href : "";
+  const url = props.url || currentUrl;
+  const shareData = { ...props, url };
   const { onCopy, hasCopied } = useClipboard(shareData.url);
 
   const toast = useToast();
```

Comment:
might as well move this to `sonner` now, right?

Reply:
I'll change this later iterations